src/UniversalRewardsDistributor.sol

```diff
@@ -96,6 +97,16 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributorStaticTyping
         _setRoot(pendingRoot.root, pendingRoot.ipfsHash);
     }
 
+    /// @notice Revokes the pending root.
+    /// @dev Can be frontrunned with `acceptRoot` in case the timelock has passed.
```

Comment:
I think that this natspec comment is not true anymore

Reply:
I think it's still valid. One can trigger `acceptRoot` as long as `block.timestamp >= pendingRoot.validAt`.